Writerside/topics/Auth-Framework.md

```diff
@@ -0,0 +1,21 @@
+# Auth Framework
+
+Most of the RPCs provided by Catena services require some form of authorization to use. For example, the call may
+require:
+
+* a session to correlate state for the user - their account, party, etc
+* elevated privileges - an admin account to delete a leaderboard entry
+* a trusted service - a game server reading match information or granting items
+
+Authorization in Catena is handled it distinct two forms:
```

Comment:
```suggestion
Authorization in Catena is handled it two distinct forms:
```

Reply:
"Authorization in Catena is handled in two distinct forms:"